drivers/at86rf215/at86rf215_netdev.c

```diff
@@ -1044,6 +1110,7 @@ static void _isr(netdev_t *netdev)
 
         if (_ack_frame_received(dev)) {
             xtimer_remove(&dev->ack_timer);
+            dev->ack_timeout = false; // we got an ACK so we can forget this timeout
```

Comment:
Should already have been set to `false` in [line 932](https://github.com/benpicco/RIOT/blob/0a16ad98d06f65e46aa9a0699842a189ee6edbe7/drivers/at86rf215/at86rf215_netdev.c#L932).

Reply:
As I stated in the commit message: there were cases where it was not and this line fixed the my "stuck in ISR" bug.

Reply:
Yes sorry, noticed this later too.